pkg/cmd/ssh/connect_information.go

```diff
@@ -200,6 +202,7 @@ func (p *ConnectInformation) String() string {
 		p.Bastion.UserKnownHostsFiles,
 		nodeHostname,
 		p.NodePrivateKeyFiles,
+		p.User,
```

Comment:
add user also adapt `NewConnectInformation` (see inline comment in suggestion)
```go
func NewConnectInformation(
	bastion *operationsv1alpha1.Bastion,
	bastionPreferredAddress string,
	bastionPort string,
	bastionUserKnownHostsFiles []string,
	nodeHostname string,
	sshPublicKeyFile PublicKeyFile,
	sshPrivateKeyFile PrivateKeyFile,
	nodePrivateKeyFiles []PrivateKeyFile,
	nodes []corev1.Node,
	user string, // <------------------------- add
) (*ConnectInformation, error) {
	// ...
	return &ConnectInformation{
		Bastion: Bastion{
			Name:             bastion.Name,
			Namespace:        bastion.Namespace,
			PreferredAddress: bastionPreferredAddress,
			Port:             bastionPort,
			Address: Address{
				IP:       bastion.Status.Ingress.IP,
				Hostname: bastion.Status.Ingress.Hostname,
			},
			SSHPublicKeyFile:    sshPublicKeyFile,
			SSHPrivateKeyFile:   sshPrivateKeyFile,
			UserKnownHostsFiles: bastionUserKnownHostsFiles,
		},
		NodeHostname:        nodeHostname,
		NodePrivateKeyFiles: nodePrivateKeyFiles,
		Nodes:               nodeList,
		User:                user, // <------------------------- add
	}, nil
}

Reply:
sure, no problem. but do we really need print os login user name there ?

Reply:
f01801b